src/ui_screensaver.c

```diff
@@ -0,0 +1,121 @@
+/* Tezos Ledger application - Screen-saver UI functions
+
+   Copyright 2024 TriliTech <contact@trili.tech>
+   Copyright 2024 Functori <contact@functori.com>
+   Copyright 2023 Ledger
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+#ifdef HAVE_BAGL
+#include "ui_screensaver.h"
+
+#include "bolos_target.h"
+#include "globals.h"
+
+#define G_screensaver_state global.dynamic_display.screensaver_state
+
+/**
+ * @brief Black screen element
+ *
+ */
+static const bagl_element_t black_screen_elements[] = {{{BAGL_RECTANGLE,
```

Comment:
black and blank , why these two variations in naming. Use blank for black_screen_elements too.

Reply:
Done:
[fixup! UX: reintroduce the custom screensaver for NANOS](https://github.com/trilitech/ledger-app-tezos-baking/pull/100/commits/76244fff61193810c9b703299ce3181616f666ed)

Rename `black` to `blank`